CHANGES.md

```diff
@@ -1,6 +1,8 @@
 unreleased
 -------------------
 
+- Allowing multiple errors to be reported in one pass of the context_free transformation.(#453, @burnleydev1)
```

Comment:
Allowing multiple error to be reported is a nice effect of this PR, but I think the best description is that raising an exception does not cancel the whole context free phase anymore!